q2_sample_classifier/tests/test_classifier.py

```diff
@@ -1148,7 +1148,7 @@ def test_metatable(self):
     def test_metatable_missing_error(self):
         with self.assertRaisesRegex(ValueError, "missing values"):
             sample_classifier.actions.metatable(
-                self.md2, missing_values='error')
+                self.md2, missing_values='error').view(pd.DataFrame)
```

Comment:
good call, not sure but I think the `view` might have been detritus from some earlier debugging.

In any case it is doing nothing here and this test "works" because the intended error is raised before attempting (and inevitably failing) to `view`. This test is ensuring that the error is raised when values are missing in the table.

Removing now, thx for catching.